backend/dal/Pims.Dal.csproj

```diff
@@ -49,7 +49,6 @@
   <ItemGroup>
     <ProjectReference Include="..\entities\Pims.Dal.Entities.csproj" />
     <ProjectReference Include="..\core\Pims.Core.csproj" />
-    <ProjectReference Include="..\notifications\Pims.Notifications.csproj" />
```

Comment:
Switched the relationship around so that it will be easier to use different email providers.